.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "overrides": [
+    {
+      "files": ["**/*.test.js", "src/helpers/**/*.js"],
+      "env": { "jest": true },
+      "plugins": ["jest"],
+      "extends": ["plugin:jest/recommended"],
+      "rules": {
+        "jest/prefer-expect-assertions": "off",
+        "no-undef": "off",
+        "no-global-assign": "off"
+      }
+    },
+    {
+      "files": ["**/*.cy.js"],
```

Comment:
You didn't find the need to put the `cypress.config.js` file in the files section here?

Reply:
Added!